src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -20,6 +20,8 @@
         <ImplicitUsings>enable</ImplicitUsings>
         <EnableDefaultCssItems>false</EnableDefaultCssItems>
 
+        <DefaultItemExcludes>$(DefaultItemExcludes);**\.DS_Store</DefaultItemExcludes>
```

Comment:
Can we do this in our targets instead of in the templates ?

Reply:
Yeah we definitely don't want this in the templates. This is an SDK type of change.

Reply:
I'll need some help:

Where do you want me to add those changes:

```
./src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.Before.targets
./src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.After.targets
./src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.targets
```

I guess

```
./src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.targets
```

Please, correct me if I am wrong (not MAUI/SDK expert)